src/components/Dashboard/FilingHistoryList/FilingTemplate.vue

```diff
@@ -0,0 +1,172 @@
+<template>
```

Comment:
This template -- with slots for override content -- is common across all filings.